src/components/Search/SearchBar.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useContext, useState } from 'react';
+import TextInput from 'ink-text-input';
+import { Text, Box, useFocus, useInput, useFocusManager } from 'ink';
+import { BrowserSessionContext } from '../../BrowserSessionProvider';
+import { SearchResults } from './SearchResults';
+
+export const SearchBar = () => {
+  const { isFocused } = useFocus({ id: 'search-bar'});
+  const { focus, focusPrevious } = useFocusManager();
+  const session = useContext(BrowserSessionContext);
+  const [inputValue, setInputValue] = useState('');
+  const [ searchResultActive, setSearchResultActive ] = useState(false);
+
+  useInput((_, key) => {
+    if (!isFocused || searchResultActive) return;
+
+    if (key.upArrow) {
+      setSearchResultActive(false);
+      focusPrevious();
+    }
+  });
+
+  const handleSearchSubmit = function (query: string) {
+    focus('search-selector');
```

Comment:
I don't see where we declare this focusable component. Is this supposed to be the search results?

Reply:
Yup, forgot to remove. Was declared in the SearchResults component but has been removed for SRState